uint8 SCAN360=0
uint8 TRACK=1
uint8 mode
string object